docs/pages/programming/jdbc_api.md

```diff
@@ -38,7 +40,23 @@ The JAR files normally can be found in `~/.m2/repository/com/bloomberg/comdb2/cd
 Another option is to build the driver inside a Docker container by running `make jdbc-docker-build` in `cdb2jdbc` (JAR files will be written
 to `cdb2jdbc/maven.m2/repository/com/bloomberg/comdb2/cdb2jdbc/2.0.0/`)
 
+#### Gradle
```

Comment:
Could you please provide the minimum version of Gradle required to build the driver using Gradle itself (not using the wrapper)?